modal_proto/api.proto

```diff
@@ -1060,6 +1062,31 @@ message TokenFlowWaitResponse {
   bool timeout = 3;
 }
 
+message VolumeCreateRequest {
```

Comment:
Proto changes could be a separate PR.

But I'm not sure there's a point to that - easier to review the proto changes with some context on how it will be used, I reckon.